app/assets/javascripts/rejection.js

```diff
@@ -0,0 +1,17 @@
+$(function() {
```

Comment:
Didn't know it existed. I am still looking to find out how to add data toggle & target to the corresponding collection value. Why is the use of javascript problematic?